app/scripts/app.js

```diff
@@ -255,32 +255,31 @@ var app = angular.module('bsis', [ // eslint-disable-line angular/di
 
       // INVENTORY URLs
       .when('/inventory', {
-        templateUrl: 'views/inventory.html',
-        controller: 'InventoryCtrl',
+        redirectTo: '/manageInventory',
         permission: PERMISSIONS.VIEW_INVENTORY_INFORMATION,
```

Comment:
@timler Just wondering about this permission and which ones we should be using for inventory. I think we can just leave it as-is for now and take a better look at the permissions when we actually implement these pages. We must just remember to test with a few different users at that point.